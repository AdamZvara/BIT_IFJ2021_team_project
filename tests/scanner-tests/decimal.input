10.5
0.8
0.0
10.0

1e5
1E5
5e+1
6E+2
1e-2
1E-2

1.5e2
1.3562E2
1.8e+2
3.5E+1
0.8e-2
300E-6

010.500
05e2
003E1