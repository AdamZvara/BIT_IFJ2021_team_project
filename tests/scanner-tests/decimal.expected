TOK_DECIMAL : 10.500000
TOK_DECIMAL : 0.800000
TOK_DECIMAL : 0.000000
TOK_DECIMAL : 10.000000
TOK_DECIMAL : 100000.000000
TOK_DECIMAL : 100000.000000
TOK_DECIMAL : 50.000000
TOK_DECIMAL : 600.000000
TOK_DECIMAL : 0.010000
TOK_DECIMAL : 0.010000
TOK_DECIMAL : 150.000000
TOK_DECIMAL : 135.620000
TOK_DECIMAL : 180.000000
TOK_DECIMAL : 35.000000
TOK_DECIMAL : 0.008000
TOK_DECIMAL : 0.000300
TOK_DECIMAL : 10.500000
TOK_DECIMAL : 500.000000
TOK_DECIMAL : 30.000000
